internal/backends/winit/renderer/sw.rs

```diff
@@ -184,4 +192,29 @@ impl super::WinitCompatibleRenderer for WinitSoftwareRenderer {
         // and the buffer age doesn't respect that, so clean the partial rendering cache
         self.renderer.set_repaint_buffer_type(RepaintBufferType::NewBuffer);
     }
+
+    fn grab_window(
+        &self,
+        window: &i_slint_core::api::Window,
+    ) -> Result<SharedImageBuffer, PlatformError> {
+        let size = window.size();
+
+        let Some((width, height)) = size.width.try_into().ok().zip(size.height.try_into().ok())
+        else {
+            // Nothing to render
+            return Err("grab_window() called on window with invalid size".into());
+        };
+
+        // We could use Surface::fetch() here, but it's only implemented on X11 and Web.
+
+        let mut target_buffer =
+            SharedPixelBuffer::<i_slint_core::graphics::Rgb8Pixel>::new(width, height);
+
+        self.force_next_frame_new_buffer.set(true);
```

Comment:
Why do you need force_next_frame_new_buffer given that you already set the buffer type in the next line? This will clear the caches.

Reply:
You're right, this isn't needed indeed. That said, I think I need to clear caches after this rendering, as the next call - from the windowing system or user - might pass an "age 1" buffer for example and caches don't apply to that one (As it's a different buffer).